Enterprise-CMCS/macpro-mdct-mcr
# run e2e a11y tests for mobile, tablet, desktop automatically

## Description

Satisfies [MDCT-398: Automatically run e2e tests at all breakpoint sizes](https://qmacbis.atlassian.net/browse/MDCT-398)

We discovered that integration tests were missing some mobile accessibility issues. This was because mobile sizes were not being tested – just desktop. Accessibility integration tests now run on all device sizes, not just desktop.

Changes:
- Add a utility to `accessibility.js` to create and run the accessibility tests for each breakpoint size.
- Call that utility in each of the 5 current route-specific accessibility .spec files.
- Tweaked cypress, axe, and pa11y configurations to match current needs.
- Fixed 2 minor accessibility issues that this change uncovered (both were cases where aria-labels needed to be added to mobile views because the descriptive text for the links/buttons is not rendered on mobile).

### How to test

- See that e2e tests now include 3 accessibility tests for each route. 
- Verify that all tests pass.
- Enjoy the 📖 

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
